app-crypt/loop-aes-losetup/loop-aes-losetup-2.34.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools toolchain-funcs flag-o-matic
+
+MY_PV="${PV/_/-}"
+MY_P="util-linux-${MY_PV}"
+LOOPAES_P="loop-AES-v3.7p"
+S="${WORKDIR}/${MY_P}"
+
+DESCRIPTION="Loop-AES losetup utility"
+HOMEPAGE="https://www.kernel.org/pub/linux/utils/util-linux/"
+SRC_URI="https://www.kernel.org/pub/linux/utils/util-linux/v${PV:0:4}/${MY_P}.tar.xz
+	http://loop-aes.sourceforge.net/loop-AES/${LOOPAES_P}.tar.bz2"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~sparc ~x86"
+
+LICENSE="GPL-2 LGPL-2.1 BSD-4 MIT public-domain"
+SLOT="0"
+IUSE="nls selinux static"
+
+RDEPEND="selinux? ( sys-libs/libselinux )"
+BDEPEND="${RDEPEND}
```

Comment:
One more question: how is libselinux used in this build system? 

If it links to .so or uses header files while building, it'd be DEPEND, but if it uses binaries from that package, it'd be BDEPEND. Seeing it's in RDEPEND I suspect it wants the .so. Could you check and update accordingly?

Reply:
ok @juippis I got a chance to come back to this: the {,B,R}DEPEND weirdness was inherited from the old version of util-linux that this ebuild was based on. I started over, updated loop-aes-losetup based on the current util-linux, and updated the PR.